build-cmakelibs.sh

```diff
@@ -38,13 +38,8 @@ git clone --depth 1 -b 0.2.5 https://github.com/yaml/libyaml || { exit 1; }
 ##
 ## Build cmake projects
 ##
-build zlib
+build zlib "-DUNIX:BOOL=ON"
 build libpng "-DPNG_SHARED=OFF -DPNG_STATIC=ON"
 build freetype
 build libyaml
 cd ..
-
-##
-## Fix legacy applications using libz.a instead of libzlib.a
-##
-ln -sf "$PS2SDK/ports/lib/libzlib.a" "$PS2SDK/ports/lib/libz.a"
```

Comment:
Isn't "libzlib" the most standard name for this library (as it is in Linux)? "libz" is what ps2dev has been using. The old solution was to name the library "libzlib", and add a symlink "libz" for compatibility with both Linux and ps2dev, making porting applications more easy.

What is the reason to change this?

Reply:
It is dependent on the POSIX standard. Libz.a is the default name for unix systems, but somehow when compiling with cmake UNIX flag isnt set properly. You can test it, when making with simple make, libz.a is provided (even with common gcc). I just fixed cmake side.

Reply:
"-DUNIX:BOOL=ON" is enough to change the library name. As ps2 is definitely UNIX style system, we can just use that flag.